### DISCUSSION TITLE
Adjustable column widths

### DISCUSSION THREAD

**Discussion by katympete** [TfTHacker/DashboardPlusPlus#3] [Q&A]
Hi there! I love this snippet, but I'm a bit of a css newbie and was wondering if you could help me with figure out something.

Is there a way to adjust the code so the width of the screen is fully occupied by the number of columns? For example, if two columns are being used (two bullet points in the syntax of the css), then each column would get 50% of the available space, rather than the fixed width that they have now? For three columns, each would get 33% of the available space, and so on and so forth. I've attached an example image that I mocked up in notion - obviously going to look different in obsidian, but I'm curious if it's even possible!

![image](https://user-images.githubusercontent.com/28572039/171086329-043e7b09-5133-4b5e-ba8a-b619aacb183e.png)

**Comment by purierca** [comment]
@katympete late to the party but love this idea!
@TfTHacker would you have any idea how to implement, simple CSS?

**Reply by purierca** [reply]
Here's a solution that I quite enjoy
https://github.com/efemkay/obsidian-modular-css-layout
